### ISSUE TITLE
Drawing page

### ISSUE THREAD

**Issue by Buliwif** [BookStackApp/BookStack#5060]
### Describe the feature you'd like

First, I want to thank you for this amazing tool, Bookstack, which I have been proudly advocating for over 5 years now.

I am a heavy user of draw.io and have created many technical diagrams with multiple tabs. I would like to integrate them into Bookstack as pages without having to go through the page editor. This way, I could quickly switch from one tab to another in my diagrams without needing to edit the page.

Do you think this is possible?

Thank you for your feedback/opinion.

Have a great day,
Arnaud

### Describe the benefits this would bring to existing BookStack users

Giving Bookstack an even more significant role in document management

### Can the goal of this request already be achieved via other means?

I don't know.

### Have you searched for an existing open/closed issue?

- [X] I have searched for existing issues and none cover my fundamental request

### How long have you been using BookStack?

Over 5 years

**Comment by Hallsie** [comment]
I agree with this.  I am newer to BookStack and Draw.io however I have been a visio user for years as well as Gliffy.  I would love to be able to dump my draw.io file into the article and it run the draw.io engine with tabs in the page.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/5060
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
